.env

```diff
@@ -43,6 +43,10 @@ PG_APPLICATION_NAME=stacks-blockchain-api
 # Limit to how many concurrent connections can be created, defaults to 10
 # PG_CONNECTION_POOL_MAX=10
 
+# Insert concurrency when processing new blocks
+# It'd better set it higher, e.g. 8 or 16, if your postgres is running on SSD and multiple CPU cores
+# INSERT_CONCURRENCY=4
```

Comment:
Please rename this to `STACKS_BLOCK_DATA_INSERT_CONCURRENCY`, also add a comment stating that this value should be set in harmony with `PG_CONNECTION_POOL_MAX` so all the concurrent writes have enough available connections to service them.

Reply:
It doesn't relate to connection pool size, the batch inserts happens within a transaction, which uses a single connection. Batching reduces interaction rounds, and sending requests in parallel reduces time to wait for responses of other irrelevant requests.

Reply:
Renamed.

Reply:
Ah, you're right, I was thinking from the read side of the API. New name is good.